### ISSUE TITLE
Improper tests

### ISSUE THREAD

**Issue by tarrenj** [DiZ73/SRIC#1]
We wont have a connection to the real internet at the comp, it'll just be a lone AP with a server attached to it.  We'll have to attempt to connect to an IP address that'll be given to us right before our flight.

**Comment by DiZ73** [comment]
I'll see what I can do. I believe that this works with the IP address. I'll
redo the tests, I can see where I went wrong there. Sorry.

**Comment by tarrenj** [comment]
No worries.  Sorry I haven't had the time to help more with this.

**Comment by DiZ73** [comment]
Any set back is my fault, I will get this done.

**Comment by tarrenj** [comment]
It might be faster to just attempt to ping the server with an os.system call.  I haven't done any tests yet, but I'm sure a ping -c 1 has got to be faster than however the hell urllib2 tests, right?

**Comment by DiZ73** [comment]
if you look at dev you'll see i've fixed the method. you just have to add <b>`http://`</b>

**Comment by tarrenj** [comment]
Right, I'm just curious as to what will take more CPU cycles, a ping test, or urllib2.

**Comment by DiZ73** [comment]
I'm not sure about that, but If you want it to run continuous, all you'd need is a single ping script. Again, I don't know.

**Comment by DiZ73** [comment]
I'll look it up, as soon as I can, I'm also looking into the SCP file system module.

**Comment by tarrenj** [comment]
We need to make sure that the competition will allow us to use SCP first. My bias is because it's a little faster, but that's useless if we're not allowed to use it...

**Comment by DiZ73** [comment]
True, true, I had a look at the SRIC you put up, and It looks good, the only question I have is that, we would need to change the IP Address. Other than that, we can easily change it.

**Comment by tarrenj** [comment]
We'll be given the IP address at the start of the test flight, so we can plug an ethernet cable into the pi, configure all of our scripts, and then pull the cable back.

We want to test connectivity using the IP of the server, since we'll always have connectivity to localhost.

**Comment by DiZ73** [comment]
In that case it looks good, the reason I created the node server was for that reason, so we could run something to actually connect to. However this was just the test. The next thing for me to do is actually read section 7.7.4 in the AUVSI Competition Rules
Once I do that, I'll have a better idea of the task, and It would be easier If we get a yes on the SCP stuff.

**Comment by tarrenj** [comment]
Also checkout appendix F.  Did you see my email on the rules?  I could use some help understanding them as well...

**Comment by DiZ73** [comment]
Yeah, that's my reason for going back to it actually, I'll also take a look at Appendix F. Will do.

**Comment by DiZ73** [comment]
I read the rules and the appendix F and I think I get how this is supposed to work. So we actually have to set up a distant station for this. It'll be held on another computer running with a modem, and the plan needs to access the computer with all its modules that we are writing and upload images and any text files that they need. We then give the judges our assigned user and pass so they can check out the files.
